File: bernardo/Problema0292/problema0292.py
# Entrada de valores
lista = []
tamanho = 10
quantidade = 0

for num in range(tamanho):
    valor = (
        float(input(f"Digite o {num + 1} número (digite 0 para finalizar): ")))
    if valor == 0:
        break
    else:
        lista.append(valor)

# Cálculo da quantidade do último número
for num in range(len(lista)):
    if lista[num] == lista[-1]:
        quantidade += 1

# Saída
print(f"Quantidade de {lista[-1]} digitados: {quantidade}")

File: bernardo/Problema0292/problema0292.js
const prompt = require("prompt-sync")({ sigint: true });

// Entrada de valores
var lista = []
var tamanho = 10
var quantidade = 0

for (var i = 0; i < tamanho; i++) {
    valor = parseFloat(prompt("Digite o " + (i + 1) + " número (digite 0 para finalizar): "));
    if (valor == 0) {
        break;
    }
    else {
        lista[i] = valor;
    }
}

// Cálculo da quantidade do último número
for (var i = 0; i < lista.length ; i++) {
    if (lista[i] == lista[lista.length - 1]) {
        quantidade++;
    }
}

// Saída
console.log("Quantidade de " + lista[lista.length - 1] + " digitados: " + quantidade);
